common/src/main/java/com/hartwig/actin/util/ApplicationConfig.java

```diff
@@ -2,13 +2,16 @@
 
 import java.io.File;
 import java.nio.file.Files;
+import java.util.Locale;
 
 import org.apache.commons.cli.CommandLine;
 import org.apache.commons.cli.ParseException;
 import org.jetbrains.annotations.NotNull;
 
 public final class ApplicationConfig {
 
+    public static final Locale locale = Locale.ENGLISH;
```

Comment:
Isn't It more standard to use capitals for static final parameters? So LOCALE rather than locale

Reply:
Yes, I'll fix this.